cli/cmd/vuln_container.go

```diff
@@ -340,6 +362,16 @@ func checkOnDemandContainerVulnerabilityStatus(reqID string) error {
 	if vulCmdState.Html {
 		return generateVulnAssessmentHTML(results)
```

Comment:
With the current implementation you have, if a user passes the `--html` flag, we won't evaluate the fail_on flags. We will need to move the evaluation of those flags somewhere else that allow the user to use all the flags available.

Reply:
mmm, I think setting the err here then returning it after would work. Question is do we want to continue with the vulnFailureValidate if the html output fails or not. I would say no, in which case the code would be.

`	cli.OutputHuman(buildVulnerabilityReportTable(results))
	if vulCmdState.Html {
		err = generateVulnAssessmentHTML(results)
	}

	if err !=nil {
		return err
	}

	if vulFailureFlagsEnabled() {
		vulnAssess := vulAssessment{
			severityRating:        results.HighestSeverity(),
			fixableSeverityRating: results.HighestFixableSeverity(),
			fixableVulCount:       results.FixableVulnerabilities,
		}
		vulnAssess.validate()
	}`

Reply:
That is better 👆🏽